dev-docs/bidders/etarget.md

```diff
@@ -44,8 +44,6 @@ In release 5.0 and later, publishers should use the `ortb2` method of setting Fi
 - ortb2.user.ext.data.*
 - ortb2.user.data[]
 
-With regards to Contextual and Audience segments, the Magnite exchange supports the IAB standard taxonomies. See [the segment management user guide](https://resources.rubiconproject.com/resource/publisher-resources/segment-management-user-guide/) for more information.
-
```

Comment:
Removing this means that your adapter no longer supports this taxonomy?

Reply:
yes we are supporting IAB standard taxonomies but we dont have link for peview so i thought that i will remove that link, but unfortunately i removed all the text. now it should be back without link.